src/main/java/org/broadinstitute/consent/http/service/ElectionService.java

```diff
@@ -426,6 +430,10 @@ public List<Election> createDataSetElections(String referenceId, Map<User, List<
         return CollectionUtils.isEmpty(electionsIds) ? null : electionDAO.findElectionsByIds(electionsIds);
     }
 
+    public List<Election> findElectionsByVoteIdsAndType(List<Integer> voteIds, String electionType) {
+        return electionDAO.findElectionsByVoteIdsAndType(voteIds, electionType);
```

Comment:
Looking at usages of this, it looks possible to pass in an empty list of vote ids. Might be safer to handle that case before making the DAO call which will throw a sql exception if it is empty.